CHANGELOG.md

```diff
@@ -43,6 +43,11 @@ and this project adheres to
   bug causing a read/write from an iovec to be duplicated when receiving an
   error on an iovec other than the first. This caused a data corruption issue in
   the vsock device starting from guest kernel 6.17.
+- [#5494](https://github.com/firecracker-microvm/firecracker/pull/5494): Fixed a
+  bug where soft lockups were detected by watchdog on microVMs restored from
+  snapshots. This moved the timing of KVM_KVMCLOCK_CTRL ioctl call, which
```

Comment:
nit: can avoid usage of word "timing" which may be read as the duration of the call:
> ... by moving the KVM_KVMCLOСK_CTRL ioctl call, ..., from after pause to before resume.

Reply:
removed "the riming"!